frontend/packages/shared/src/components/LoadingProgressIndicator/LoadingProgressIndicator.tsx

```diff
@@ -1,25 +1,43 @@
-import { FC, PropsWithChildren } from 'react';
+import { FC, PropsWithChildren, useRef, useEffect } from 'react';
 
 import { CustomCircularProgress } from '../CustomCircularProgress/CustomCircularProgress';
 import './LoadingProgressIndicator.scss';
 
+const usePreviousValue = (value: boolean) => {
+  const ref = useRef<boolean>();
+  useEffect(() => {
+    ref.current = value;
+  });
+  return ref.current;
+};
+
 interface LoadingProgressIndicatorProps {
   isLoading: boolean;
   displayBlock?: boolean;
+  translateCommon?: (t: string) => string
 }
 
 export const LoadingProgressIndicator: FC<
   PropsWithChildren<LoadingProgressIndicatorProps>
-> = ({ isLoading, displayBlock, children }) => {
+> = ({ isLoading, displayBlock, translateCommon, children }) => {
   const classSuffix = displayBlock ? '__block' : '__inline-flex';
+  const prevIsLoading = usePreviousValue(isLoading);
 
   return (
     <div className={`loading-progress-indicator${classSuffix}`}>
       <div className={`loading-progress-indicator__container${classSuffix}`}>
         {children}
         <div className="loading-progress-indicator__container__spinner-box">
           {isLoading && (
-            <CustomCircularProgress size={'3rem'} color={'secondary'} />
+            <CustomCircularProgress
+              title={translateCommon && translateCommon('loadingContent')}
+              aria-live="assertive"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions
sanoo seuraavaa: `aria-live="assertive" should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention.`

Päteekö tän PR:n scopeen?

Reply:
Emma varmaan osaisi vastata parhaiten, mutta kopioin tuon ajatuksen täältä: https://carbondesignsystem.com/components/loading/accessibility/